experimental/client-app/package.json

```diff
@@ -45,9 +43,8 @@
     "@web/rollup-plugin-html": "^1.11.0",
     "eslint": "^8.22.0",
     "eslint-config-prettier": "^8.5.0",
-    "husky": "^8.0.1",
+    "husky": "^4.3.8",
```

Comment:
Is this downgrade/rollback intentional?

Reply:
Correct downgrade was intentional. Had been erroring out for me at `8.0.1` so I downgraded to `4.3.8`. Just  did further digging and just needed to remove `.git/hooks/` locally (https://github.com/npm/cli/issues/1845). Reinstating 8.0.1.